packages/microfrontend-sdk/src/MicrofrontendIntegrator.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  Collections,
+  Endpoints,
+  IEnvironment,
+  IProject,
+  PublicVariable,
+  ServiceConfigMap,
+  Services,
+} from '@mia-platform-internal/console-types'
+import { QiankunProps } from 'vite-plugin-qiankun/dist/helper'
+import { Subject } from 'rxjs'
+
+import { Events } from './types'
+
+type IConfigObservable = {
+  endpoints: Endpoints
+  collections: Collections
+  configMaps: ServiceConfigMap
+  services: Services
+  unsecretedVariables: PublicVariable[]
+
+  // TOFIX: add correct fast data type
+  fastDataConfig: Record<string, unknown>
```

Comment:
I would not provide this in the SDK, at least in the first iteration let's remove it and strive towards the migration of fast data to the new integration framework

Reply:
removed